state-chain/pallets/cf-broadcast/src/migrations.rs

```diff
@@ -7,5 +7,5 @@ mod migrate_timeouts;
 pub type PalletMigration<T, I> = (
 	VersionedMigration<Pallet<T, I>, initialize_broadcast_timeout_storage::Migration<T, I>, 6, 7>,
 	VersionedMigration<Pallet<T, I>, migrate_timeouts::Migration<T, I>, 7, 8>,
-	PlaceholderMigration<Pallet<T, I>, 8>,
```

Comment:
why this change?

Reply:
@syan095

Reply:
We have `SerializeSolanaBroadcastMigration` in the Runtime crate that migrates broadcaster pallets from version 8 -> 9, so I've updated the "placeholder" migration (which should always points to the current version).

I've updated this with a comment to make it clearer

Reply:
I see. I was under the impression that the placeholder migration should point to the latest version after update, but seems like we don't need to. 

This change was overridden by a merge from main.

Reply:
UPDATE: the placeholder version should match the latest pallet's storage version, or it will give out a warning on try-runtime.
I've updated this to version 10 (as 10 is the latest version after merge from main)

Reply:
Update again: Looks like with version 10, we get a warning when doing the pallet migrations, but we are OK in the post-migration checks. 
If we leave the version at 8, we are OK when doing pallet migrations, but will get a warning in post-migration checks.

this is due to how we do the migrations - we do pallet migrations first, then the Runtime ones.

It's just warnings, so I guess I'll just leave it at version 10, since that's the version the pallets ended up on after all the migrations are done.